.github/workflows/publish-pr-preview.yml

```diff
@@ -9,6 +9,8 @@ on:
     paths:
       - '.github/actions/build-and-publish-pr-preview/**/*'
       - '.github/actions/build-setup/**/*'
+      - '.github/actions/build-pr-site/**/*'
+      - '.github/workflows/build-all-site.yml'
```

Comment:
The current worklfow as no relation with this action and this workflow, so I suggest we remove them from the path